src/actions/value.js

```diff
@@ -40,22 +40,20 @@ export function resetValuesToDefault() {
 
 		let valueToSet;
 		Object.keys(selectedValues).forEach((component) => {
-			if (
-				!componentProps[component]
-				|| !componentProps[component].defaultValue
-				|| !componentProps[component].componentType
-			) {
+			if (!componentProps[component] || !componentProps[component].componentType) {
```

Comment:
@mohdashraf010897 The default case is to set the `value` to `null`. So if any of the above conditions is `true` then set the value to `null`.